src/libcharon/plugins/kernel_super/system_events/sl_system_events/sl_system_events.h

```diff
@@ -0,0 +1,20 @@
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "../system_events_interface.h"
+
+typedef struct sl_system_events_obj_t sl_system_events_obj_t;
```

Comment:
Remove the "obj" part from all the names in this PR, there is no "object" that is being created here.
(If you're curious the "object" in liboper_db_client is the "DB object" which represents what fields are to be modified by the `liboper_db_client`. There is no such thing for system_events)

Reply:
Lack of creativity for names, seems odd to just call it "sl_system_events_t".

Anyway, I changed it, thanks